src/app/core/admin/admin-entity-list/admin-entity-list.component.html

```diff
@@ -4,15 +4,22 @@
 
 <div>
   <div
-    cdkDropList
-    cdkDropListOrientation="horizontal"
-    (cdkDropListDropped)="drop($event, config?.filters)"
```

Comment:
I think the `drop()` method is not used anymore then. If so, please remove it from the component.ts also

Reply:
oh, right! Thanks